File: ExamenMoria-Python/projecto/models/salasPython/SalaAccion.py
from Sala import Sala

class SalaAccion(Sala):
    def __init__(self, id: int, flechas: int, enemigos: int):
        super().__init__(id)
        self.flechas = flechas
        self.enemigos = enemigos

    def __str__(self) -> str:
        return f"SalaAccion({self.numeroSala}, {self.flechas}, {self.enemigos})"

File: ExamenMoria_Java/src/main/java/models/salas/SalaAccion.java
package models.salas;

public class SalaAccion extends Sala{

    public int flechas;

    public int enemigos;

    public SalaAccion(int numeroSala, int flechas, int enemigos) {
        super(numeroSala);
        this.enemigos = enemigos;
        this.flechas = flechas;
    }

    @Override
    public String toString() {
        return "SalaAccion{" +
                "flechas=" + flechas +
                ", enemigos=" + enemigos +
                ", numeroSala=" + numeroSala +
                '}';
    }
}
